plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergStatistics.java

```diff
@@ -50,6 +51,7 @@ protected QueryRunner createQueryRunner()
     {
         return IcebergQueryRunner.builder()
                 .setInitialTables(NATION)
+                .setIcebergProperties(ImmutableMap.of("iceberg.file-format", "ORC"))
```

Comment:
Document why. We should prefer to test with defaults, so any non-default setup should be documented, unless it's obviosu.

Reply:
The assertions in this class were not written to expect non-null values for data size since that is not implemented for ORC. I've now updated the assertions with data sizes instead of this change.